### PR TITLE
Gmail to python 3

### PR DESCRIPTION
**Pull request by sapirshuker**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-3782](https://jira-hq.paloaltonetworks.local/browse/CIAC-3782)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by Shellyber — CHANGES_REQUESTED** [review]
Great job! 👍 
Few general notes: 
1. Regarding the test and the input_data file - for short examples (for example user_id_test or title) please move it to the unittest file itself. Only long and full responses should be in the test_data file, the rest in the unittest file
2. Update the release notes - write the changes made in the PR (without the tests) moving to python3 and fixing the bug

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 1]
```diff
@@ -429,13 +429,6 @@ def mail_to_incident(msg, service, user_key):
     return incident
 
 
-def organization_format(org_list):
```
[the remark this answers is not in the document]
Comment (sapirshuker) [reply Packs/Gmail/Integrations/Gmail/Gmail.py:432]: it does not have any use in the integration

File: Packs/Gmail/ReleaseNotes/1_2_0.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### Gmail
+- add unit test emails_to_entry, mailboxes_to_entry, filters_to_entry, sent_mail_to_entry,
```
[the remark this answers is not in the document]
Comment (sapirshuker) [reply Packs/Gmail/ReleaseNotes/1_2_0.md]: it was deleted in the last commit

File: Packs/Gmail/ReleaseNotes/1_2_0.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Gmail
+- Updated the Docker image to: *demisto/google-api-py3:1.0.0.35176*.
+- Converted the Integration to python3.
+- Fixed an issue with ***search all mailboxes***, which printed mailboxes without any results for the query.
```
Comment (Shellyber) [inline Packs/Gmail/ReleaseNotes/1_2_0.md:6]:
```suggestion
- Fixed an issue with ***search all mailboxes*** command, which printed mailboxes without any results for the query.
```

File: Packs/Gmail/Integrations/Gmail/Gmail_test.py [hunk 4]
```diff
@@ -228,7 +232,259 @@ def test_labels_to_entry():
...
+
+    from Gmail import tokens_to_entry
+
+    result = tokens_to_entry(title, response)
+    assert result.readable_output == expected_result.get("expected_human_readable")
+    assert result.outputs == expected_result.get("expected_outputs")
+    assert result.raw_response == expected_result.get('expected_raw_response')
+
+
+title_sent_mail_to_entry = 'Email sent:'
```
Comment (Shellyber) [inline Packs/Gmail/Integrations/Gmail/Gmail_test.py:389]: Use this directly in the parametrize section (below)
No need to declare it here and then use it
Same goes for all

File: Packs/Gmail/Integrations/Gmail/Gmail_test.py [hunk 5]
```diff
@@ -228,7 +232,259 @@ def test_labels_to_entry():
...
+    from Gmail import sent_mail_to_entry
+
+    result = sent_mail_to_entry(title, response, to, emailfrom, cc, bcc, body, subject)
+    assert result.readable_output == expected_result.get("expected_human_readable")
+    assert result.outputs == expected_result.get("expected_outputs")
+    assert result.raw_response == expected_result.get('expected_raw_response')
+
+
+title_test_filters_to_entry = 'filters:'
+mailbox_test_filters_to_entry = '111111111111111111111'
```
Comment (Shellyber) [inline Packs/Gmail/Integrations/Gmail/Gmail_test.py:420]: Let's shorten the example here (and in the test_data)

**Review by Shellyber — COMMENTED** [review]
Nice job! 
Last notes! Also don't forget to fix the build

File: Tests/conf.json [hunk 6]
```diff
@@ -5202,6 +5198,7 @@
         }
     ],
     "skipped_tests": {
+        "get_original_email_-_gmail_-_test" : "Uses a deprecated playbook called get_original_email_-_gmail",
```
Comment (Shellyber) [inline Tests/conf.json:5201]: Need to remove this

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 7]
```diff
@@ -578,15 +562,14 @@ def user_roles_to_entry(title, response):
...
-        'HumanReadable': tableToMarkdown(title, context, headers, removeNull=True),
-        'EntryContext': {'Gmail.Role(val.ID && val.ID == obj.ID)': context}
-    }
+    human_readable = tableToMarkdown(title, context, headers, removeNull=True)
+    return CommandResults(
+        raw_response=context,
+        outputs=context,
+        readable_output=human_readable,
+        outputs_prefix='Gmail.Role',
+        outputs_key_field=['ID']
```
Comment (Shellyber) [inline Packs/Gmail/Integrations/Gmail/Gmail.py:571]:
```suggestion
        outputs_key_field='ID'
```

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 8]
```diff
@@ -653,17 +635,16 @@ def role_to_entry(title, role):
...
-        'ReadableContentsFormat': formats['markdown'],
-        'HumanReadable': details_hr + privileges_hr,
-        'EntryContext': {'Gmail.Role(val.ID && val.ID == obj.ID)': context}
-    }
+    return CommandResults(
+        raw_response=context,
+        outputs=context,
+        readable_output=details_hr + privileges_hr,
+        outputs_prefix='Gmail.Role',
+        outputs_key_field=['ID']
```
Comment (Shellyber) [inline Packs/Gmail/Integrations/Gmail/Gmail.py:646]:
```suggestion
        outputs_key_field='ID'
```

**Review by Shellyber — APPROVED** [review]
Great job. See my last comments before merge

**Pull request merged**